00-locals.tf

```
@@ -25,16 +25,23 @@ locals {
 
   common_config_patch = {
     cluster = {
+    
       id          = var.cluster_id,
       clusterName = var.cluster_name,
       apiServer = {
-        certSANs = [
-          module.elb_k8s_elb.elb_dns_name
-        ]
+        extraArgs = {
+          cloud-provider = "external"
+          enable-admission-plugins = "MutatingAdmissionWebhook,ValidatingAdmissionWebhook, ServiceAccount"
```

```suggestion
          enable-admission-plugins = "MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ServiceAccount"
```

& please make that configurable.